usage/camera/raspicam/v4l2.md

```diff
@@ -0,0 +1,13 @@
+## Using the V4L2 driver
+
+The V4L2 driver provides a Linux standard driver for accessing camera features. This is the driver needed to use the Raspberry Pi camera range as, for example, webcams.
+
+The V4L2 driver provides a standard API over the firmware-based camera system.
+
+
+### Installing the V4L2 driver
+
+Installation of the V4L2 driver is automatic. It is loaded as a child of the VCHIQ driver, and once loaded it will check how many cameras are attached and then create the right number of device nodes.
+
+
+
```

Comment:
Would it be worth suggesting some example apps in which the V4L2 driver can be used?

Reply:
Yes, but not sure of the best ones to suggest. @6by9 has suggested Qv4l2 so will put that in for the moment.